### DISCUSSION TITLE
Elite Pi, and the list of supported controller

### DISCUSSION THREAD

**Discussion by iamim** [Yowkees/keyball#396] [Q&A]
Just ordered my keyball39 from yushakobo to Norway and can't wait to start building it. In the rush I forgot to buy the controllers and now have to order them here in Europe. Unfortunately, ProMicro are hard to get at the moment but this one promises full compatibility with ProMicro boards: https://splitkb.com/collections/keyboard-parts/products/elite-pi. As an added bonus, it has more memory which seems to be useful for trackball builds.

Is Elite Pi currently supported? An I likely to have issues completing the build with it?

**Comment by Yowkees** [comment]
Hi iamim,

Thank you very much for choosing Keyball39.

If you want to use Elite Pi, you should learn about PRK Firmware.
Please check hasumikin's repositories.
https://github.com/hasumikin

He is great programmer and he is using Keyball44 by PRK Firmware.
I hope you could get good advice from his project.

Thank you.

**Reply by iamim** [reply]
Hi @Yowkees ,

Thanks for the reply! I'll have a look at the repo.
To make it simple in the beginning, will I be able to flush Elite Pi or other "ProMicro-compatible" controllers with the firmware that you have in the build guide?
